plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduConnectorTest.java

```diff
@@ -148,6 +189,55 @@ public void testRowDelete()
         assertUpdate("DROP TABLE test_row_delete");
     }
 
+    @Test(dataProvider = "testColumnNameDataProvider")
+    @Override
+    public void testColumnName(String columnName)
+    {
+        if (!requiresDelimiting(columnName)) {
+            testColumnName(columnName, false);
+        }
+        testColumnName(columnName, true);
+    }
+
+    private void testColumnName(String columnName, boolean delimited)
```

Comment:
This is a general deficiency in our base tests today - they cannot be adapted to fit different CREATE TABLE/CTAS syntax. The connectors which might benefit from this are ClickHouse, Druid, Kudu and Phoenix IIRC.

Reply:
I think we can make it `protected` or extract one line for creating a table. Let me handle in another PR because other tests also has the similar issue.